src/lib/viewers/media/DashViewer.js

```diff
@@ -387,6 +408,25 @@ class DashViewer extends VideoBaseViewer {
         }
     }
 
+    /**
+     * Handles streaming event which is the first time the manifest is available. See https://shaka-player-demo.appspot.com/docs/api/shaka.util.Error.html
```

Comment:
Update these docs. Looks like you're no longer handling/accepting a `shakaError` param